lib/render.js

```diff
@@ -132,27 +132,33 @@ function getVis() {
 }
 
 function renderLater() {
-    clearTimeout(reloadTimer);
-    reloadTimer = setTimeout(function() {
-        renderChart();
-    }, 100);
+    _.defer(renderChart);
 }
 
-function initResizeHandler(vis, container) {
-    const resize = getHeightMode() === 'fixed' ? resizeFixed : renderLater;
-    let curWidth = width(container);
+function initResizeHandler() {
+    let currentWidth = width($chart);
 
-    // IE continuosly reloads the chart for some strange reasons
-    if (navigator.userAgent.match(/msie/i) === null) {
-        window.onresize = resize;
-    }
```

Comment:
was this tested? removing an IE edge case `if` without replacement rings alarm bells for me

Reply:
we decided to not go with the `ResizeObserver` for now but to still get rid of this specific hack.